runtime/mount/python_lib/lplots/widgets/button.py

```diff
@@ -88,8 +96,13 @@ def w_button(
             "default": default,
             "readonly": readonly,
         },
-        _signal=_state.use_value_signal(key=key),
+        _signal=_state.use_value_signal(key=f"{key}"),
+        _trigger_signal=_state.use_value_signal(key=trigger_key),
     )
     _emit.emit_widget(key, res._state)
 
+    asyncio.run_coroutine_threadsafe(
```

Comment:
this is a bit scary cause you could deadlock

Reply:
but ig not much we can do about it

tho maybe we make the button widget specifically async idk

add a todo to figure this out for now - probably won't do anything about it if it's just one widget